src/__tests__/Button.test.js

```diff
@@ -120,6 +120,13 @@ describe('<Button>', function() {
     expect(ambManager.toggleMenu).toHaveBeenCalledTimes(1);
   });
 
+  test('prevent behavior of a Button element when the space key is pressed', function() {
+    const wrapper = shallow(el(Button, null, 'foo'), shallowOptions);
+    wrapper.simulate('keyUp', spaceEvent);
+
+    expect(spaceEvent.preventDefault).toHaveBeenCalledTimes(1);
```

Comment:
I think we can improve upon this assertion here. Rather than asserting `event.preventDefault` is called (that doesn't really provide any userful information whether the menu is being opened/toggled), we should first open the menu, and simulate the `keyDown`, and `keyUp` with the `spaceEvent` you have and check if `toggleMenu` is called multiple times [like so](https://github.com/davidtheclark/react-aria-menubutton/blob/5b88ff9459b8c1f6543186ae2c98a8dca7a30868/src/__tests__/Button.test.js#L115).

Reply:
Thanks.
Could you tell me if you have a good idea about this?
In the test case, I think it is difficult to reproduce the behavior of Firefox simulating a `click` event when you release the space bar on a `button` element.
Calling `toggleMenu` is the role of `keyDown` or `click`, and `keyUp` itself is not involved in calling it, hence it asserts whether keyUp calls `event.preventDefault`.

Reply:
Yeah, I'm not sure how to run these tests against a FF environment (I think this can be tackled as a separate issue). However, the tests is just to ensure it doesn't cause any regression in general.

I think something like the steps below would work:

1. simulate click to open menu first
2. expect `ambManager.isOpen` is `true` (ensures the menu is open before we run assertions to validate the <kbd>Space</kbd> events
3. simulate `keyDown` and then a `keyUp` <kbd>Space</kbd> event
4. expect `ambManager.isOpen` is `false`

Reply:
Thank you very much.
I will work on it.